docs/getting-started/quick-start.md

```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
 
 import 'ckeditor5/dist/styles.css';
 
-ClassicEditor
+await ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		plugins: [ Essentials, FormatPainter, SlashCommand ],
 		toolbar: [ 'formatPainter' ],
 		licenseKey: 'your-license-key'
 	} )
-	.catch( err => {
-		console.error( err );
-	} );
+```
+
+## Using CKEditor&nbsp;5 from CDN
+
+CDN is an effortless method of running CKEditor 5. You can start using it in just a few steps and with a few tags.
+
+### Obtain a license key
+
+To activate CKEditor&nbsp;5 premium features, you will need a commercial license. If you wish to purchase a commercial CKEditor&nbsp;5 license or a license to one of the premium features, [contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs. To obtain an activation key, please follow the {@link support/license-key-and-activation License key and activation} giuide.
+
+<!-- Not sure we should duplicate this in QS: TODO: Describe the steps to obtain a license key, attaching some screenshots. -->
+
+### CDN setup
+
+Start by attaching a link to the style sheet. It contains all styles for the editor's UI and content. You can also include your styles if you like. Refer to the content styles guide for more information.
+
+```html
+<link rel="stylesheet" href="<CDN_LINK>/ckeditor5/dist/styles.css" />
+```
+
+Then, you need to attach the script with the JavaScript code. To simplify imports, you can use the feature available in browsers &ndash; the [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). It maps the text used as the module specifier and the value that will replace the text when resolving the specifier. This way, import statements do not need to contain the full CDN URL.
+
+```html
+<script type="importmap">
+	{
+		"imports": {
+			"ckeditor5": "<CDN_LINK>/ckeditor5/dist/index.min.js",
+			"ckeditor5/": "<CDN_LINK>/ckeditor5/"
```

Comment:
@filipsobol is this always needed?

Reply:
It's only needed when importings translations, but I'm fine with having this here.